examples/memo/src/lib.rs

```diff
@@ -0,0 +1,40 @@
+use concordium_std::*;
+
+/// # Implementation of a smart contract that can receive transfers with a memo
+/// message and forward them to the owner account.
+/// All this contract does is expose a single `receive` method which checks that
+///
+/// - it is being invoked by an account
+/// - the message it is receiving is 32 bytes long
+///
+/// And if both of these are valid it forwards the amount it received to the
+/// owner account.
+
+#[derive(Serialize, SchemaType)]
+/// The contract has no initialization parameters.
+struct InitParameter;
+
+/// Init function that creates a new contract.
+#[init(contract = "memo", parameter = "InitParameter", low_level)]
```

Comment:
You could save a few bytes by not adding the parameter to the schema, assuming you are embedding the schema.

Reply:
Yes, but I wanted to add it for symmetry.